README.rst

```diff
@@ -45,7 +45,7 @@ With a few steps, you can start in your client repository using CI confiurations
   script:
     - .ci_config/travis.sh
 
-* Note that `.ci_config` is the REQUIRED name of the cloned folder; it is hardcoded on `industrial_ci` side so you must adhere to it.
+* Note: The name `.ci_config` is NO longer REQUIRED for the cloned folder since version 0.3.2; you can name it whatever you like.
```

Comment:
catkin does not follow hidden folders, so it is basically a `CATKIN_IGNORE`.
(I don't want to `CATKIN_IGNORE` `industrial_ci` as it is a valid catkin package, which should be released later on)
If the CI folder is not hidden, it gets included in the build (and the logs), which is not that bad.
As an alternative the CI folder might be placed outside of `TARGET_REPO_PATH`, e.g. in `/tmp/' oder '~/'.